src/proj_4D_api.c

```diff
@@ -342,16 +342,22 @@ PJ  *proj_create_crs_to_crs (PJ_CONTEXT *ctx, const char *srid_from, const char
         PJ *P = proj_create_crs_to_crs(0, "epsg:25832", "epsg:25833", NULL);
 
 ******************************************************************************/
-    PJ *P;
+    PJ *P = NULL;
     char buffer[512];
 
     /* area not in use yet, suppressing warning */
     (void)area;
 
+    if (!strchr(srid_from, ':') || !strchr(srid_to, ':')) {
+        proj_log_error(P, "input does not appear to be a valid SRID");
+        return 0;
+    }
+
     strcpy(buffer, "+proj=pipeline +step +init=");
     strncat(buffer, srid_from, 512-strlen(buffer));
     strncat(buffer, " +inv +step +init=", 512-strlen(buffer));
     strncat(buffer, srid_to, 512-strlen(buffer));
+    printf("%s\n", buffer);
```

Comment:
Leftover debugging code?

Reply:
Yes. Should not have been included in this commit.